### DISCUSSION TITLE
Yandex Navigator

### DISCUSSION THREAD

**Discussion by im0-ru** [BimmerGestalt/AAIdrive#734] [Ideas]
Thanks for the development!
Is it possible to connect Yandex Navigator for use on the BMW screen? It has an Android Auto app.

**Comment by hufman** [comment]
As far as I know, there isn't a general way to embed an application (such as Yandex) into another app (AAIdrive), which would be needed to screenshot the app without the screen on and show it in the car. Android Auto works because it has a special API into these apps, but that API is restricted to only be used by Android Auto.
